packages/webapp/src/pages/Integrations/providerConfigKey/Endpoints/components/One.tsx

```diff
@@ -0,0 +1,241 @@
+                            </h3>
+                            <div className="flex flex-col gap-5">
+                                {body.fields.map((field) => {
+                                    return (
+                                        <div key={field.name} className="flex flex-col pb-5 gap-2.5 border-b border-b-border-gray last-of-type:border-b-0">
+                                            <div className="flex justify-between">
+                                                <div className="flex gap-2">
+                                                    <code className="font-code text-text-light-gray text-s">{field.name}</code>
+                                                    <code className="font-code text-text-light-gray text-s bg-dark-600 px-2 rounded-md">
+                                                        {/* {'value' in field ? (Array.isArray(field.value) ? 'Arr' : field.value) : field.type} */}
+                                                        {'value' in field ? fieldToTypescript({ field }) : field.type}
+                                                    </code>
+                                                </div>
+                                                {'optional' in field && field.optional && <div className="text-text-light-gray text-s">Optional</div>}
+                                            </div>
+                                        </div>
+                                    );
+                                })}
+                            </div>
+                        </div>
+                    )}
+
+                    {!queryParams && !metadata && !body && <div className="text-text-light-gray px-5 italic">No parameters or body</div>}
+                </div>
+                <div className="flex flex-col grow gap-10 w-1/2">
+                    <div className="flex flex-col border border-active-gray rounded-md">
+                        <header className="flex justify-between items-center bg-active-gray px-4 py-2 rounded-t-md">
+                            <div className="text-text-light-gray">Request</div>
+                            <div className="flex gap-2 items-center ">
+                                <Select defaultValue={language} onValueChange={(v) => setLanguage(v as any)}>
+                                    <SelectTrigger className="bg-dark-600 uppercase text-text-light-gray text-[11px] px-1.5 py-0.5 h-auto">
+                                        <SelectValue placeholder="Language" />
+                                    </SelectTrigger>
+                                    <SelectContent>
+                                        <SelectItem value="node" className="text-s">
+                                            Node
+                                        </SelectItem>
+                                        <SelectItem value="curl" className="text-s">
+                                            Curl
```

Comment:
cURL

Reply:
it's confusing wikipedia says cURL but everywhere in the official doc and repo it's curl 😮‍💨
I'll set cURL for consistency 👌🏻